app/models/entry.rb

```diff
@@ -53,11 +53,13 @@ def calculate(type, *args)
 
       private
       def all(*args)
-        find_many :find, :all, *args
+        ActiveSupport::Deprecation.warn('Passing arguments is deprecated') if args.any?
+        find_many :all # *args
```

Comment:
@ulferts this works. I'm not sure why the various levels of indirection here though. Could this not be simplified?